src/com/solvd/travelagency/ICheckIn.java

```diff
@@ -0,0 +1,7 @@
+package com.solvd.travelagency;
+
+public interface ICheckIn {
+    void checkIn();
+
+    void allowIn();
+}
```

Comment:
be consistent in formatting
```suggestion
public interface ICheckIn {
    void checkIn();
    void allowIn();
}
```

Reply:
Code reformatted. Please check.